CONTRIBUTING.md

```diff
@@ -23,6 +23,24 @@ Note for Windows: Use the latest version of [Git Bash](https://gitforwindows.org
 
 1. Use [TablePlus](https://tableplus.com/) for managing sqlite database fixtures.
 
+1. We use the [Jest test framework](https://jestjs.io/). Its CLI is powerful and removes the need for npm scripts mostly. For most cases this is what you need to know:
+
+   ```
+   yarn jest <fileNamePattern> -t <testNamePattern>
+   ```
+
+1. Some integration tests in these packages use [Jest's `each` feature](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout). If you only want to run a subset of the test cases, simply leverage the `-t` flag on the command line (see above point). For example in `packages/cli` here is how you would run Just the `findOne where PK` cases for sqlite integration:
+
+   ```
+   yarn jest integrate.sqlite -t 'findOne where PK'
+   ```
+
+   Also you can piggy back flags onto existing npm scripts. For example the above could be rewritten as:
+
+   ```
+   yarn test:sqlite -t 'findOne where PK'
```

Comment:
I am suggesting `yarn` here because `pnpx` and `pnpm` do not seem to support the same featureset.